streamer/src/nonblocking/quic.rs

```diff
@@ -950,104 +950,102 @@ async fn handle_connection(
     let stable_id = connection.stable_id();
     stats.total_connections.fetch_add(1, Ordering::Relaxed);
     while !stream_exit.load(Ordering::Relaxed) {
-        if let Ok(stream) =
+        let Ok(stream) =
```

Comment:
Is this purely done to reduce the indentation? Of so, I would like to skip it to reduce the scope of the changes and focus on the real issue being solved.

Reply:
dropped indentation commit